UpdatePodSpec.py

```diff
@@ -0,0 +1,58 @@
+import json
+import base64
+import requests
+import os
+import netrc
+
+import re
+
+acversion = ''
+
+# get AdaptiveCard Version from custom.prop
+with open('custom.props', 'r') as f:
+    lines = f.readlines();
+    for line in lines:
+        if 'NuGetPackVersion' in line:
+            p = re.compile(r'\d+\.\d+\.\d+')
+            m = p.search(line)
+            if m != None:
+                acversion = m.group()
```

Comment:
should we break once we find the line containing the version?

Reply:
you're absolutely correct, thanks! 

---
In reply to: [225654731](https://github.com/Microsoft/AdaptiveCards/pull/2039#discussion_r225654731) [](ancestors = 225654731)